cmd/broker/suite_test.go

```diff
@@ -926,8 +926,12 @@ func fixConfig() *Config {
 		Broker: broker.Config{
 			EnablePlans: []string{"azure", "trial", "aws", "own_cluster", "preview", "sap-converged-cloud", "gcp", "free"},
 			Binding: broker.BindingConfig{
-				Enabled:       true,
-				BindablePlans: []string{"aws", "azure"},
+				Enabled:              true,
+				BindablePlans:        []string{"aws", "azure"},
+				ExpirationSeconds:    100,
```

Comment:
Could we change the value to 600? That was the lowest possible value for Token Requests.